puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_develop.sh

```diff
@@ -44,3 +43,11 @@ bundle exec rake db:drop db:create db:migrate --trace
 tasks="pkg:generate_source jenkins:unit"
 [ ${database} = postgresql ] && tasks="$tasks jenkins:integration"
 bundle exec rake $tasks TESTOPTS="-v" --trace
+
+# Test asset precompile
+if [ ${database} = postgresql -a -e "$APP_ROOT/package.json" ]; then
+  bundle exec rake db:drop db:create RAILS_ENV=production
+  bundle exec rake db:migrate RAILS_ENV=production
+  bundle exec rake assets:precompile RAILS_ENV=production
+  bundle exec rake webpack:compile RAILS_ENV=production
```

Comment:
Should this also be `NODE_ENV=production` so we get the production bundle built?

Reply:
Yes, https://github.com/mipearson/webpack-rails/blob/a6faf5d2fc1a76481ddf640fb799245d719a7951/lib/tasks/webpack.rake#L5 does this.